test/zdtm/static/netns_lock.hook

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+import subprocess
+import socket
+import sys
+import os
+
+from ctypes import CDLL
+
+libc = CDLL('libc.so.6')
+
+CLONE_NEWNET = 0x40000000
+PORT = 8880
+NETNS = "criu-net-lock-test"
+TIMEOUT = 0.1
+
+def nsenter():
+    f = open("/var/run/netns/{}".format(NETNS))
+    libc.setns(f.fileno(), CLONE_NEWNET)
+    f.close()
```

Comment:
Python has a [`with`](https://www.python.org/dev/peps/pep-0343/) statement that could handle exceptions to ensure closing the file
```py
with open("/var/run/netns/{}".format(NETNS)) as f:
    libc.setns(f.fileno(), CLONE_NEWNET)
```